### PR TITLE
Created quickstart

### PR DESCRIPTION
**Pull request by rikutakei**
I thought the README was too long so I created a separate markdown for a TL;DR.

I've also adjusted the TranslatorGuidelines.md so each line is a single sentence that ends with 「。」.
This is purely just for my preference (and now that I think about it, I probably shouldn't have changed it...)

### CODE REVIEW

**Comment by joelnitta** [comment]
Thanks for catching the extra spaces in the guidelines. I think the quickstart looks great! Tom do you want to merge since the PR is mostly about the README, which you were working on most recently?

File: quickstart.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+```
+git clone git@github.com:<github_username>/i18n.git
+```
+4. Add remote so you can pull changes from original repo:
+```
+git remote add ja git@github.com:swcarpentry-ja/i18n.git
+```
+5. Make changes to your local repo - make sure to commit often:
+```
+git add *
```
Comment (TomKellyGenetics) [inline quickstart.md]:
```suggestion
git add -u
```

I would advise this to update files and avoid adding extra files. Of course we should put anything we don't want added into the `.gitignore` as well. #26

**Comment by TomKellyGenetics** [comment]
Looks great. This is exactly what we need if we're getting people less confident with git involved. Actually, I was thinking to make the README shorter and move the more detailed sections to separate files. I'm happy to merge this in the meantime. Just a minor suggestion to change.

**Comment by rikutakei** [comment]
Added your suggested change @TomKellyGenetics - merging the PR

**Pull request merged**